src/seismometer/_api.py

```diff
@@ -148,14 +157,12 @@ def target_feature_summary(exclude_cols: list[str] = None, inline=False):
 
 
 @export
-def fairness_audit(metric_list: Optional[list[str]] = None, fairness_threshold=1.25) -> None:
+def fairness_audit(metric_list: Optional[list[str]] = None, fairness_threshold=1.25) -> HTML:
```

Comment:
same thing: typehint for `metric_list` does not match docstring

Reply:
FINC